package.json

```
@@ -152,6 +152,12 @@
             "description": "Output additional diagnostics to the Swift Output View.",
             "order": 8
           },
+          "swift.SDK": {
```

```suggestion
          "swift.advanced.SDK": {
```
This is an advanced feature that most users of the extension will not use so can we flag it as such. "advanced" flags will get grouped together.

---

I guess this is not so useful in solving the problem.

However, it costs too high for us to move a config from Advanced to non-Advanced. It’s also not so good to hard-code a config as advanced.

Can we break the configurations into groups like CodeLLDB does? The grouped configurations can be adjusted at any time without breaking any user settings, and it’s much more obvious than a plain "prefix" like `Swift > Advanced: …`.

---

We could break the configuration into groups like Code-LLDB. I previously didn't because it required a later version of VSCode. But the next release of vscode-swift will require 1.65.0 so we can move to using groups. Shall we leave that to a later PR which breaks off an advanced section and a Sourcekit-LSP section

---

I think we should do this by 0.5.0 release, so mark as resolved now:)